source/linux/Foundational_Components/Power_Management/pm_low_power_modes.rst

```diff
@@ -18,6 +18,7 @@ Texas Instruments has added support for the following low power modes:
 #. Deep Sleep
 #. MCU Only
 #. Partial I/O
+#. I/O Only Plus DDR
```

Comment:
I am wondering if we should sort these from lowest latency to highest latency just to be consistent?

Reply:
```suggestion
#. MCU Only
#. Deep Sleep
#. I/O Only Plus DDR
#. Partial I/O
```

Reply:
I think this is a good idea if we explain that order in the above paragraph.

Reply:
We can list the ordering from lowest latency to highest latency separately but the following sections where these modes are explained are interdependent. e.g
MCU Only = Deep sleep + MCU core
I/O DDR = Partial I/O + DDR 
Will have to rewrite the entire section.
So, adding a separate table

Reply:
@akashdeep-ti  please can you ping once you add this separate table?
Thanks.

Reply:
Updated in order of power consumption